libs/application/templates/general-petition/src/hooks/useHasEndorsed.ts

```diff
@@ -18,5 +18,7 @@ export const useHasEndorsed = (endorsementListId: string) => {
       },
     },
   )
-  return endorsement?.endorsementSystemUserEndorsements?.hasEndorsed
+
+  console.log(endorsement)
```

Comment:
is this console log required for future development?

Reply:
Console.log slipped through?